apps/backoffice-v2/src/lib/blocks/hooks/useDocumentBlocks/useDocumentBlocks.tsx

```diff
@@ -388,9 +388,32 @@ export const useDocumentBlocks = ({
                         value,
                         formatMinimum,
                         formatMaximum,
+                        default: defaultValue,
                       },
                     ]) => {
-                      const fieldValue = value || (properties?.[title] ?? '');
+                      const getFieldValue = () => {
+                        if (typeof value !== 'undefined') {
+                          return value;
+                        }
+
+                        if (
+                          typeof properties?.[title] === 'undefined' &&
+                          typeof defaultValue !== 'undefined'
+                        ) {
+                          return defaultValue;
+                        }
+
+                        if (typeof properties?.[title] === 'undefined' && type === 'boolean') {
+                          return false;
+                        }
```

Comment:
Can be worthwhile to map other types as well such as returning 0 for type number and so on.